third_party/2and3/click/decorators.pyi

```diff
@@ -31,7 +31,7 @@ def make_pass_decorator(
 
 def command(
     name: Optional[str] = ...,
-    cls: type = Command,
+    cls: Optional[Type[Command]] = ...,
```

Comment:
This one is Optional, unlike the others in this file: https://github.com/pallets/click/blob/master/click/decorators.py#L92.